spec.html

```diff
@@ -8720,7 +8724,7 @@ <h1>FunctionDeclarationInstantiation ( _func_, _argumentsList_ )</h1>
               1. Else,
                 1. Let _initialValue_ be ! _envRec_.GetBindingValue(_n_, *false*).
               1. Call _varEnvRec_.InitializeBinding(_n_, _initialValue_).
-              1. NOTE: vars whose names are the same as a formal parameter, initially have the same value as the corresponding initialized parameter.
+              1. NOTE: Vars whose names are the same as a formal parameter initially have the same value as the corresponding initialized parameter.
```

Comment:
"vars" seems right to me; if capitalized i'd expect "Variables". not entirely sure why.

Reply:
Yeah, I wasn't totally sure. It seems "vars" is only used three times total and this is the only sentence-initial case. [13.3.2](https://tc39.es/ecma262/#sec-variable-statement) does have "Var variables" at the beginning of a sentence though. In general, we seem to be inconsistent with regard to whether backticks are used, e.g. "var declaration" vs. "`var` declaration", but it's only really noticeable when you'd otherwise expect capitalization.

Reply:
As a matter of fact, the other two instances of "vars" are from the very same algorithm, in this note which appears twice:
> NOTE: Only a single lexical environment is needed for the parameters and top-level vars.

I think if we're going to use "vars" like this then it should be fine to capitalize it, but another approach would be to make the note here singular, such that it'd start with "A var whose...".